.github/workflows/stale-message.yml

```diff
@@ -0,0 +1,29 @@
+name: "Close stale issues and PRs"
+on:
+  schedule:
+    - cron: "30 1 * * *"
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4
+        with:
+          stale-issue-message: 'Old issues will be closed after 105 days of inactivity. This issue has been quiet for 90 days and is being marked as stale. Reply here or add the label "not-stale" to keep this issue open.'
```

Comment:
I don't know if all user can add labels, so we may just ask for a comment here.

Reply:
i feel like the user that initiated the PR can usually add labels but i'll find out for sure and if not we can get rid of that suggestion

Reply:
hmm didn't actually find any definitive answer about PRs but i did see some for labels not being available. we do say _or_  add a label, do you think that's clear enough or would you rather we completely remove it?

Reply:
yeah we had just written in the message that you could either comment or add a not stale label to de-stale the issue/PR. but we can change the language to not tell folks to add a label